lib/matplotlib/backend_bases.py

```diff
@@ -2169,7 +2172,7 @@ def print_figure(
 
                 # call adjust_bbox to save only the given area
                 restore_bbox = _tight_bbox.adjust_bbox(
-                    self.figure, bbox_inches, self.figure.canvas.fixed_dpi)
+                    self.figure, bbox_inches, renderer, self.figure.canvas.fixed_dpi)
```

Comment:
Passing the renderer here does not affect the target issue but I assume that passing it is more efficient than potentially calling `_get_renderer` again.